backend/app/services/leave_company_service.rb

```
@@ -0,0 +1,52 @@
+      end
+    end
+
+  def remove_user_roles!
+    user.company_workers.where(company: company).delete_all
+    user.company_investors.where(company: company).delete_all
+    user.company_lawyers.where(company: company).delete_all
+  end
+
+  def user_is_administrator?
+    user.company_administrators.exists?(company: company)
+  end
```

RuboCop (1.76.1)

[convention] Inconsistent indentation detected.

(Layout/IndentationConsistency)